### ISSUE TITLE
PyQT issue with clean install on OSX El Capitan

### ISSUE THREAD

**Issue by Cumol** [MouseLand/suite2p#119]
Trying to install suite2p on OSX El Capitan and getting issues with PyQT

`Could not find a version that satisfies the requirement PyQt5 (from suite2p==0.4.4) (from versions: )
No matching distribution found for PyQt5 (from suite2p==0.4.4)`

This happens with `pip install suite2p` or using `pip install .` with the git-cloned version.

Installing pyqt5 via homebrew worked but did not resolve the issue.

**Comment by carsen-stringer** [comment]
This happened recently when I was helping some install too. The only thing that worked was making a conda environment for suite2p and installing there. Have you tried that already?

**Comment by Cumol** [comment]
Just tried it. Doesn't work

Any other ideas?

**Comment by roschkoenig** [comment]
Been installing both pip install suite2p; and github versions recently on Mac OS 10.14.1 Mojave without any issues - pip freeze tells me I've got the following versions: 
PyQt5==5.11.3
PyQt5-sip==4.19.13
Perhaps try and force install those?

**Comment by carsen-stringer** [comment]
Did that fix work? If so, please close the issue. I've added a help wanted flag in case it's still a problem.

**Comment by Cumol** [comment]
@carsen-stringer it did not. But I am starting to suspect that it is a more complex problem due to me lagging behind on el capitan. I will upgrade my system to high sierra soon and retry again.

I will close the issue for now

**Comment by kmmwalker** [comment]
I have this problem as well on my mac. None of the fixes suggested seem to solve the issue for me.

**Comment by carsen-stringer** [comment]
I've reopened this issue, do you have an older mac OS? The "help wanted" tag is activated, so maybe some other mac users have other advice and will contribute! can you post your error message and the versions of pyqt5 you have installed?

**Comment by nzanepro** [comment]
as of today, on el capitan, this works for python3 (not python2.x)

`pip install PyQt5==5.10.1`

note you might need to call pip3 depending on how your machine is set up.

`pip3 install PyQt5==5.10.1`

I came across this issue while searching on how to easily get pyqt5 installed into python2.7 on el capitan. I resorted to PySide2 on el capitan for python2.7 for my needs with an unrelated project.  i'm not familiar with this project so I can't verify that this will fully resolve the problem. hi!! and good luck!!
